tasks/qemu.py

```diff
@@ -154,6 +159,78 @@ def download_image(ctx, config):
                     ],
                 )
 
+
+def _setup_nfs_mount(remote, client, mount_dir):
+    """
+    Sets up an nfs mount on the remote so that the guest
+    can move it's logs over to the remote after the tests
```

Comment:
Also, the guest should be writing logs directly to the nfs mount, not moving them over afterward. That way if the guest blows up, we still have logs.